lib/game_manager.rb

```diff
@@ -32,4 +36,9 @@ def find_games_by_team(team_id)
     end
   end
 
+  def find_game_ids_of_season(season_id)
+    @games.find_all do |game|
+      game.season == season_id
+    end.map {|game| game.game_id}
```

Comment:
Like how clean this is.

Needs tests for `worst_coach` and `find_game_ids_of_season` in `game_manager_test.rb`